frontend/src/app/(routes)/governance/ProposalOverviewDeposit.tsx

```diff
@@ -0,0 +1,233 @@
+                      <CustomPieChart
+                        value={23}
+                        color="#759BE5"
+                        label="Turn out"
+                      />
+                      <div className="proposal-text-extralight">
+                        23% Turn Out
+                      </div>
+                    </div>
+                    <div className="flex items-center gap-2">
+                      <CustomPieChart
+                        value={50}
+                        color="#B373CA"
+                        label="Threhold"
+                      />
+                      <div className="proposal-text-extralight">
+                        50% Threshold
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="voting-grid">
+            <div className="voting-view">
+              <div className="status-pass">
+                <div className="flex flex-col items-center space-y-2">
+                  <div className="flex space-x-2 ">
+                    <Image
+                      src="./vote-icon.svg"
+                      width={20}
+                      height={20}
+                      alt="Vote-Icon"
+                    />
+                    <p className="proposal-text-small">Total Votes</p>
+                  </div>
+
+                  <p className="proposal-text-big">123,345,876</p>
+                </div>
+              </div>
+              <div className="w-full text-white flex flex-col justify-center items-center space-y-2">
+                <div>Quorum</div>
+
+                <div className="bg-white w-full h-[10px] rounded-full">
+                  <div
+                    className={
+                      `bg-[#2DC5A4] h-[10px] rounded-l-full ` + `w-[${quorum}%]`
+                    }
+                  ></div>
+                </div>
+              </div>
+
+              <div className="flex justify-between items-start gap-2">
+                <div className="flex items-center gap-2">
+                  <CustomPieChart value={75} color="#4AA29C" label="Yes" />
+                  <div className="proposal-text-extralight">75% Yes</div>
+                </div>
+                <div className="flex items-center gap-2">
+                  <CustomPieChart value={23} color="#E57575" label="No" />
+                  <div className="proposal-text-extralight">23% No</div>
+                </div>
+                <div className="flex items-center gap-2">
+                  <CustomPieChart value={2} color="#EFFF34" label="Veto" />
+                  <div className="proposal-text-extralight">2% Veto</div>
+                </div>
+                <div className="flex items-center gap-2">
+                  <CustomPieChart value={0} color="#EFFF34" label="Veto" />
+                  <div className="proposal-text-extralight">0% Veto</div>
+                </div>
```

Comment:
Render these using map().
Create an array of objects. Each object contains the vote type and percentage.